File: Trabalho_Teorico_1/Unidade_00g/exercicio_12.java
/**
 * Marcos Antonio Lommez Candido Ribeiro
 * 77157
 * AED's 2
 * 13/08/2022
 */
class exercicio_12 {
    /*
     * Faca um programa que leia a nota de 5 alunos e mostre na tela a media
     * das mesmas
     */
    public static void main(String[] args) {
        MyIO.print("Digite a nota dos alunos\n");
        int i = 0;
        float media = 0;
        while (i < 5) {
            MyIO.print("->");
            media += MyIO.readFloat();
            i++;
        }
        media /= i;
        MyIO.print("A media e: ");
        MyIO.print(media);
    }
}

File: Trabalho_Teorico_1/Unidade_00m/exercicio_12.c
/**
 * Marcos Antonio Lommez Candido Ribeiro
 * 77157
 * AED's 2
 * 14/08/2022
 */

#include <stdlib.h>
#include <stdio.h>
#include <math.h>

/** 
 * Faca um programa que leia n numeros inteiros, armazene-os em um
 * arquivo, leia-os do arquivo e mostre-os na tela
 */
int main(){
    printf("Escolha quantos numeros escrever:\n->");
    int n;
    scanf("%d", &n);
    printf("\nEscreva os numeros:\n");
    FILE* arquivo = fopen("exemplo.txt", "w");
    for(int i = 0; i < n; i++){
        int tmp;
        printf("[%d]->", i);
        scanf("%d", &tmp);
        fflush(stdin);
        fprintf(arquivo, "%d\n", tmp);
    }
    fclose(arquivo);
    printf("Arquivo salvo com sucesso\n");
    arquivo = fopen("exemplo.txt", "r");
    for(int i = 0; i < n; i++){
        int tmp;
        fscanf(arquivo, "%d", &tmp);
        printf("\n%d", tmp);
    }

    fclose(arquivo);

    return 0;
}